types/tx.go

```diff
@@ -193,39 +208,72 @@ func ComputeProtoSizeForTxs(txs []Tx) int64 {
 	return int64(pdData.Size())
 }
 
-// UnwrapMalleatedTx attempts to unmarshal the provided transaction into a malleated
-// transaction wrapper, if this can be done, then it returns true. A malleated
-// transaction is a normal transaction that has been derived (malleated) from a
-// different original transaction. The returned hash is that of the original
-// transaction, which allows us to remove the original transaction from the
-// mempool. NOTE: protobuf sometimes does not throw an error if the transaction
-// passed is not a tmproto.MalleatedTx, since the schema for PayForMessage is kept
-// in the app, we cannot perform further checks without creating an import
+// UnwrapMalleatedTx attempts to unmarshal the provided transaction into a
+// malleated transaction. It returns true if the provided transaction is a
+// malleated transaction. A malleated transaction is a transaction that contains
+// a MsgPayForBlob that has been wrapped with a share index.
+//
+// NOTE: protobuf sometimes does not throw an error if the transaction passed is
+// not a tmproto.MalleatedTx, since the protobuf definition for MsgPayForBlob is
+// kept in the app, we cannot perform further checks without creating an import
 // cycle.
 func UnwrapMalleatedTx(tx Tx) (malleatedTx tmproto.MalleatedTx, isMalleated bool) {
```

Comment:
We use `Unwrap....` here and `Unmarshal....` below for a procedure that is seemingly the same. Would prefer to converge on a single term (for me personally `Unmarshal` and `Marshal` since `Tx` is just bytes).

Reply:
:+1: https://github.com/celestiaorg/celestia-core/pull/893/commits/3058efd92135bc97b3ca532b5a01f8833df39a2e